### PR TITLE
Added RatRig Printer Profiles

### PR DESCRIPTION
**Pull request by erebus04**
I have added the RatRig printer profiles.
V-Core 3 200/300/400/500
V-Minion

### CODE REVIEW

**Comment by SoftFever** [comment]
@erebus04 Thanks for the PR.

Currently, there is a known z-fighting issue when a bed model is used.
I plan to fix it when I got time.
Until then, we can leave the `bed_model` property blank.

with bed_model:
image

without bed_model:
image

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 200.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Core 3 200",
+    "model_id": "V-Core_3_200",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Core",
+    "bed_model": "ratrig-vcore-bed-200.stl",
```
Comment (SoftFever) [inline resources/profiles/Ratrig/machine/RatRig V-Core 3 200.json:8]: bed_model creates z-fighting issues.
Let's avoid using them.

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 300.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Core 3 300",
+    "model_id": "V-Core_3_300",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Core",
+    "bed_model": "ratrig-vcore-bed-300.stl",
```
Comment (SoftFever) [inline resources/profiles/Ratrig/machine/RatRig V-Core 3 300.json:8]: bed_model creates z-fighting issues.
Let's avoid using them.

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 400.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Core 3 400",
+    "model_id": "V-Core_3_400",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Core",
+    "bed_model": "ratrig-vcore-bed-400.stl",
```
Comment (SoftFever) [inline resources/profiles/Ratrig/machine/RatRig V-Core 3 400.json:8]: bed_model creates z-fighting issues.
Let's avoid using them.

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 500.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Core 3 500",
+    "model_id": "V-Core_3_500",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Core",
+    "bed_model": "ratrig-vcore-bed-500.stl",
```
Comment (SoftFever) [inline resources/profiles/Ratrig/machine/RatRig V-Core 3 500.json:8]: bed_model creates z-fighting issues.
Let's avoid using them.

File: resources/profiles/Ratrig/machine/RatRig V-Minion.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Minion",
+    "model_id": "V-Minion",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Minion",
+    "bed_model": "ratrig-vminion-bed.stl",
```
Comment (SoftFever) [inline resources/profiles/Ratrig/machine/RatRig V-Minion.json:8]: bed_model creates z-fighting issues.
Let's avoid using them.

**Review by SoftFever — CHANGES_REQUESTED** [review]
Overall looks awesome. Thanks!

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 300.json (hunk 2 above)
Comment (erebus04) [reply resources/profiles/Ratrig/machine/RatRig V-Core 3 300.json]: The bed model was removed due to a known z bug

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 400.json (hunk 3 above)
Comment (erebus04) [reply resources/profiles/Ratrig/machine/RatRig V-Core 3 400.json]: The bed model was removed due to a known z bug

File: resources/profiles/Ratrig/machine/RatRig V-Core 3 500.json (hunk 4 above)
Comment (erebus04) [reply resources/profiles/Ratrig/machine/RatRig V-Core 3 500.json]: The bed model was removed due to a known z bug

File: resources/profiles/Ratrig/machine/RatRig V-Minion.json (hunk 5 above)
Comment (erebus04) [reply resources/profiles/Ratrig/machine/RatRig V-Minion.json]: The bed model was removed due to a known z bug

**Comment by erebus04** [comment]
The bed model was removed due to a known z bug

**Comment by erebus04** [comment]
The bed model was removed due to a known z bug

**Review by SoftFever — APPROVED** [review]

**Pull request merged**